src/index.html

```diff
@@ -6,9 +6,42 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="header__inner">
+        <a href="#home">
+          <img
+            src="/images/logo.png"
+            alt="logo"
+            class="img__logo"
+          >
+        </a>
+
+        <nav class="header__menu">
+          <ul>
+            <li><a href="#" class="is-active">Apple</a></li>
```

Comment:
```suggestion
            <li>
              <a href="#" class="is-active">Apple</a>
            </li>
```
Каждый тег с новой строки